bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -134,6 +134,12 @@ impl Room {
         self.inner.alt_aliases().iter().map(|a| a.to_string()).collect()
     }
 
+    /// Get the user who created the invite, if any.
+    pub async fn inviter(&self) -> Option<RoomMember> {
+        let invite_details = self.inner.invite_details().await.ok()?;
+        invite_details.inviter.and_then(|m| m.try_into().ok())
```

Comment:
Don't we want to log anything in the failure cases?

Reply:
Fair point, I went one step further and exposed the underlying errors to the final client to handle accordingly.